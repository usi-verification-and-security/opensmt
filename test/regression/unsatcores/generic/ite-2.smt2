(set-option :produce-unsat-cores true)
(set-logic QF_UF)
(declare-const A Bool)
(declare-const B Bool)
(declare-const C Bool)
(assert (! (= A B) :named n1))
(assert (! (ite C (not (=> A B)) (not (=> B A))) :named n2))
(check-sat)
(get-unsat-core)
