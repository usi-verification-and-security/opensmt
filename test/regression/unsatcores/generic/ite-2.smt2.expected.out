unsat
(
n1
n2
)
